bedrock/mozorg/redirects.py

```diff
@@ -222,6 +222,12 @@ def decider(request, **kwargs):
     redirect(r'^legal/eula/firefox-2/?$', '/about/legal/eula/firefox-2/'),
     redirect(r'^legal/eula/firefox-3/?$', '/about/legal/eula/firefox-3/'),
 
+    # Bug 1209643
+    redirect(r'^legal/bylaws_amendment_(?P<n>[12])(\.html|/)?', '/foundation/documents/bylaws-amendment-{n}/'),
```

Comment:
Hmm. Maybe it would be a nice feature to allow substitution from source url to view name as well... Not now, but that seems like it might come in handy for things like this.